client/js/render.js

```diff
@@ -195,13 +196,19 @@ function renderChannelUsers(data) {
 }
 
 function renderNetworks(data, singleNetwork) {
+	const networks = JSON.parse(storage.get("networks")) || {};
+	data.networks.forEach((network) => networks[network.uuid] = networks[network.uuid] || {collapsed: false, uuid: network.uuid});
```

Comment:
Change it to store just collapsed UUIDs instead of all of them, then you won't need to do defaults here.

Reply:
Fixed